src/day_05/iteration.test.js

```diff
@@ -0,0 +1,85 @@
+describe('DAY 5: Test Iteration - While', () => {
+
+    it(`(while loop)
+        condition: should loop until a is less than 5
+        body: post increment a on each iteration`, () => {
+        let a = 0;
+
+        expect(a).toBe(5);
```

Comment:
shouldn't change `toBe` to `toBeLessThan`?

Reply:
The 0 is on purpose?, because it will stop on first iteration

Reply:
if you translate the instructions to code, the answer is there
```javascript
while(a<5){
a++;
}
```